src/5e-SRD-Features.json

```diff
@@ -8812,5 +8812,19 @@
       "If you want to cast either spell at a higher level, you must expend a spell slot as normal."
     ],
     "url": "/api/features/signature-spell"
+  },
+  {
+    "index": "shelter-of-the-faithful",
+    "background": {
+      "index": "acolyte",
+      "name": "Acolyte",
+      "url": "/api/backgrounds/acolyte"
+    },
+    "name": "Shelter of the Faithful",
+    "desc": [
+      "As an acolyte, you command the respect of those who share your faith, and you can perform the religious ceremonies of your deity. You and your adventuring companions can expect to receive free healing and care at a temple, shrine, or other established presence of your faith, though you must provide any material components needed for spells. Those who share your religion will support you (but only you) at a modest lifestyle.",
+      "You might also have ties to a specific temple dedicated to your chosen deity or pantheon, and you have a residence there. This could be the temple where you used to serve, if you remain on good terms with it, or a temple where you have found a new home. While near your temple, you can call upon the priests for assistance, provided the assistance you ask for is not hazardous and you remain in good standing with your temple."
+    ],
+    "url": "/api/features/shelter-of-the-faithful"
```

Comment:
Per conversations in Discord, since backgrounds always have on feature and that feature is unique to the background, it makes more sense to bundle this into repo. And it also answers the thing that was tickling me about this addition.